### PR TITLE
PR Labeler

### PR DESCRIPTION
**Pull request by mitchdenny**
PR Labeler listens to updates to pull requests and based on the files associated with each one it applies zero or more labels. It uses the ```CODEOWNERS``` file to drive this automation.

### CODE REVIEW

File: tools/pull-request-labeler/Azure.Sdk.Tools.PullRequestLabeler/.gitignore [hunk 1]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (weshaggard) [inline tools/pull-request-labeler/Azure.Sdk.Tools.PullRequestLabeler/.gitignore:1]: Lets avoid adding this if possible.

Comment (mitchdenny) [reply tools/pull-request-labeler/Azure.Sdk.Tools.PullRequestLabeler/.gitignore:1]: Yep ... will remove. This whole PR may be able to be abandoned depending on where Alex goes with using office bot to process label assignments.

**Comment by AlexGhiondea** [comment]
I have #753 up that will allow us to create the office bot configuration task.

**Comment by mitchdenny** [comment]
Yep, I'm going to abandon this branch. Thanks for the quick solution Alex!

**Pull request closed without merging**